packages/headless/esbuild.js

```diff
@@ -13,6 +13,7 @@ const useCaseEntries = {
   'product-recommendation': 'src/product-recommendation.index.ts',
   'product-listing': 'src/product-listing.index.ts',
   'case-assist': 'src/case-assist.index.ts',
+  insight: 'src/insight.index.ts',
```

Comment:
Quotes are not required because `insight` is a valid JavaScript property name. That's why `product-recommendation`, `product-listing`, and `case-assist` have to be surrounded by quotes.